packages/flutter_tools/analysis_options.yaml

```diff
@@ -2,3 +2,11 @@
 # the ones from above, which include the `public_member_api_docs` rule).
 
 include: ../../analysis_options.yaml
+
+analyzer:
+  exclude:
+    - test/**
```

Comment:
Is this still needed now that `package:test` has distinguished `expect()` from `expectLater()`?

Reply:
Ah sure. The lints in tests are for a variety of other random reasons. Patched those as well